### DISCUSSION TITLE
Documenting fixes

### DISCUSSION THREAD

**Discussion by lewisblake** [metno/pyaerocom#630] [Ideas]
First of all, I have started a [repository](https://github.com/lewisblake/pyadevdocs) for storing documentation related to the development of pyaerocom. It is currently private, but could be made public if there is sufficient interest. There I started a discussion, which actually might be better suited for discussing at our next development meeting. Pasted below:

It could be a good idea to document (perhaps briefly) fixes associated with pyaerocom so that if similar issues arise in the future, some record of actions taken and how the fix was implemented is also described in plain language.

Some recent examples include:

- the fix for the missing module issue (related to the proj package). In the EMEP meeting today, David said he could not install pyaerocom with conda because of this issue. Daniel said "we could write a book about this issue". Maybe not a whole book is needed, but maybe a short markdown.
- the vrmo3max fix. What was the issue? What was the fix? Was it merged? etc.

Let me know what you think. I don't want to create unnecessary work for us obviously, but I'm genuinely curious if this would be helpful and if you think for certain things documenting fixes could be worth it. Not sure if it would be redundant with [the Discussions](https://github.com/metno/pyaerocom/discussions)

**Comment by avaldebe** [comment]
the discussion about the Cartopy/Proj issues is on #583,
this and was addressed by #620
The new pip install intructions are in the [docs](https://github.com/metno/pyaerocom/blob/main-dev/docs/install.rst#via-pip), not sure why are not on RTD yet.
